File: Gurpreetkour9am/TabsExample/app/src/main/java/com/example/tabsexample/ViewPagerAdapter.java
package com.example.tabsexample;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

import com.example.tabsexample.fragments.ChatFragment;
import com.example.tabsexample.fragments.StatusFragments;
import com.example.tabsexample.fragments.UserFragment;


@SuppressWarnings("deprecation")
public class ViewPagerAdapter extends FragmentPagerAdapter {

    public ViewPagerAdapter(@NonNull FragmentManager fm) {
        super(fm);
    }

    @NonNull
    @Override
    public Fragment getItem(int position) {
        switch (position)
        {
            case 0:
                return new UserFragment();
            case 1:
                return new ChatFragment();
            case 2:
                return new StatusFragments();
            default:
                return null;
        }

    }

    @Override
    public int getCount() {
        return 3;
    }
}

File: Rahoul3pm/TabLayouts/app/src/main/java/com/example/tablayouts/ViewPagerAdapter.kt
package com.example.tablayouts

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter
import com.example.tablayouts.fragements.ChatFragment
import com.example.tablayouts.fragements.SettingFragment
import com.example.tablayouts.fragements.UserFragment

class ViewPagerAdapter(var fragmentManager: FragmentManager):FragmentPagerAdapter(fragmentManager) {

    override fun getCount(): Int {
       return 3
    }

    override fun getItem(position: Int): Fragment {
        when(position)
        {
            0->{
                return UserFragment()
            }
            1->{
                return ChatFragment()
            }
            2->{
                return SettingFragment()
            }
            else->{
                return UserFragment()
            }
        }
    }
}